.github/workflows/pipeline.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           ${{ startsWith(matrix.os, 'windows') && format('build\tests\cpp\{0}\yaramod_tests.exe', matrix.config) || 'build/tests/cpp/yaramod_tests' }}
       - name: Python Tests
         run: |
-          python -m pip install pytest
+          python -m pip install pytest mypy==0.991
```

Comment:
Nitpick: Looks like we are using `requirements.txt` in the repo root for dev dependencies.

Can you also specify this dependency there. Dependabot will be able to update that in the future (once we set it up). And we will be less likely to forget about it, that it is required by tests.

Reply:
It looks like it is not used here though, otherwise I don't understand why the `python -m pip install pytest` was there :thinking:

Reply:
You are right, the `pytest` in the requirements.txt file is also locked to some ancient version. It will be a good candidate for cleanup, based on the `git blame` it could be an old remnant from Travis CI. Resolved.